lab1.js

```diff
@@ -109,6 +132,8 @@ var nextAnimal;
 
 // your code goes here
 
+nextAnimal = Math.random(favoriteAnimals);
```

Comment:
this will get you a random number, and so the assert passes - because nextAnimal is truthy. However, you're not selecting a random element from the array. You need to end up with something like 
nextAnimal[i] (that's the index of the array)
where i is a random index (in this case, 0, 1, 2, or 3).